vtex-apps/my-account
# Magiclink feature 🧙

#### What did you change? \*

- Return to the origin url passing the newly created addressId as argument.

#### Why? \*

- Enable the MagicLink feature.

#### How to test it? \*

- [Login](https://storev1--recorrenciaqa.myvtex.com) with `assinaturasqa@mailinator.com`
- [Go to](https://storev1--recorrenciaqa.myvtex.com/account#/addresses/new?returnUrl=/subscriptions/6FEA11DCB0197FAC3F46B3836EA15903)
- Create a new address
- Check if it was added to the URL through the argument `newAddressId`

#### Related to / Depends on?

- Enables:
https://github.com/vtex/my-subscriptions/pull/81

#### Types of changes \*
